engines/dfc_provider/app/controllers/dfc_provider/api/v0/base_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
```

Comment:
Ok, this is a bit nasty, as I am namespacing now the controller with API version, it is showing that the files are changing totally with is not true. Basically just changed indentation.